### PR TITLE
fits 0.8.6-1

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
:+1:

**Comment by bfontaine** [comment]
Seems the bot didn’t even test the formula because it depends on Java 1.7 and the bot doesn’t have it:

```
+ brew test-bot --ci-pr
==> git checkout efc8043						 PASSED
==> brew pull --clean https://github.com/Homebrew/homebrew/pull/40434	 PASSED
==> brew doctor								 PASSED
==> brew --env								 PASSED
==> brew config								 PASSED
==> brew tests								 PASSED
==> brew readall							 PASSED
==> brew uses fits							 PASSED
==> SKIPPING: fits
Java 1.7 is required to install this formula.
You can install with Homebrew Cask:
  brew install Caskroom/cask/java

You can download from:
  http://www.oracle.com/technetwork/java/javase/downloads/index.html
==> git checkout -f master						 PASSED
==> git reset --hard							 PASSED
==> brew cleanup --prune=30						 PASSED
```

File: Library/Formula/fits.rb [hunk 1]
```diff
@@ -1,11 +1,9 @@
-require 'formula'
-
 class Fits < Formula
   desc "File Information Tool Set"
-  homepage 'http://fitstool.org/'
-  url 'http://projects.iq.harvard.edu/files/fits/files/fits-0.8.4_0.zip'
-  version '0.8.4'
-  sha1 'f9b6b13cf1e818c6cdcfec71eb3dcd6804dd8819'
+  homepage "http://projects.iq.harvard.edu/fits"
+  url "http://projects.iq.harvard.edu/files/fits/files/fits-0.8.6_1.zip"
```
Comment (vszakats) [inline Library/Formula/fits.rb]: This `url` does support https (without redirection):
https://projects.iq.harvard.edu/files/fits/files/fits-0.8.6_1.zip

The `homepage` as well, but it redirects to plaintext.

Comment (bfontaine) [inline Library/Formula/fits.rb]: Thanks, I fixed the `url`.

**Comment by mikemcquaid** [comment]
:+1:

**Comment by bfontaine** [comment]
Seems it was a problem on my side since the bot happily built it with Java 1.8.

**Pull request closed without merging**